Lib/_pyrepl/main.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
I don't really understand the purpose of `main.py` here. Can you explain that, please, @pablogsal?

Reply:
This was a leftover from the previous status of the PR before I started cleaning it. I will remove this file, sorry